pkg/config/config.go

```diff
@@ -285,14 +292,14 @@ type ModConnection struct {
 // Returns true if a connection does not functionally link the outputs and
 // inputs of the modules. This can happen when a module is connected with "use"
 // but none of the outputs of fromID match the inputs of toID.
-func (mc *ModConnection) isEmpty() (isEmpty bool) {
-	isEmpty = false
+func (mc *ModConnection) isUnused() bool {
+	var isUnused bool
```

Comment:
Can you one-line it? 
`return mc.Kind == useConnection && len(mc.sharedVariables) == 0`